src/firebolt/service/V2/types.py

```diff
@@ -53,6 +53,10 @@ class EngineStatus(Enum):
     def __str__(self) -> str:
         return self.value
 
+    @classmethod
+    def from_string(cls, value: str) -> "EngineStatus":
+        return cls(value.capitalize())
```

Comment:
I'm just worried that now we have two ways of instantiating this class: `EngineStatus()` and `EngineStatus.from_string()` that work differently. Should this go into `__init__` instead?